src/newcharacter.cpp

```diff
@@ -2420,7 +2420,7 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
     for( const auto &loc : start_locations::get_all() ) {
         if( get_scenario()->allowed_start( loc.id ) ) {
             uilist_entry entry( loc.id.get_cid().to_i(), true, -1, string_format(
-                                    ngettext( "%s (<color_white>%d</color> variant)",
+                                    ngettext( "%s ",
```

Comment:
```suggestion
                                    ngettext( "%s",
```
Probably shouldn't be a trailing space here.

Reply:
Ah, I see the space was added as the hack.  That doesn't make much sense...

Reply:
This actually won't work for languages without the singular/plural distinction, since these languages can only have one translated string what represents both the singular and plural forms. You should instead add a special case for the one variant case.

Reply:
And I think "%s" doesn't work because only the first string parameter of `ngettext` is used as the translation id, so it simply returns the translation for "%s" (which is, presumably, "%s" too).

Reply:
Ah, so this doesn't do an actual evaluation if it's just 1 or more. That actually makes sense. Although it makes it worse that it works in English. I assume that's because there is no actual English translation but instead English is the id for all the other translations?

Reply:
Yes, the strings are returned as-is if you don't compile a translation file for English.

Reply:
Huh, I always assumed that the translation key for `ngettext` was the pair of strings, not just the first one, but I guess it makes sense that it wouldn't work that way.